src/clients/ntp-client_proposal.rb

```diff
@@ -362,10 +351,10 @@ def Write(param)
       return :success if param["write_only"]
 
       # One-time adjusment without running the ntp daemon
-      # Meanwhile, ntpdate was replaced by sntp
+      # Meanwhile, ntpdate was replaced by sntp and later by chrony
```

Comment:
well it is nice to know what ntp tools used to be used in suse but it belongs into chronicle not here

Reply:
fair enough, will remove it.